lib/node_modules/@stdlib/math/base/special/rad2degf/README.md

```diff
@@ -0,0 +1,200 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# rad2degf
+
+> Convert an angle from radians to degrees (single-precision).
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var rad2degf = require( '@stdlib/math/base/special/rad2degf' );
+```
+
+#### rad2degf( x )
+
+Converts an angle from radians to degrees (single-precision).
+
+```javascript
+var d = rad2degf( 3.141592653589793 / 2.0 );
+// returns 90.0
+
+d = rad2degf( -3.141592653589793 / 4.0 );
+// returns -45.0
+
+d = rad2degf( NaN );
+// returns NaN
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   Due to finite precision, canonical values may not be returned. For example,
+
+    ```javascript
+    var d = rad2degf( 3.141592653589793 / 6.0 );
+    // returns 29.999999999999996
```

Comment:
```suggestion
    // returns 30.000001907348633
```

Reply:
I would keep this in mind, to re-calculate values for `float32` versions, as they might be different than those for `float64`.